src/commons/editor/EditorContainer.tsx

```diff
@@ -1,24 +1,61 @@
+// Necessary to prevent "ReferenceError: ace is not defined" error.
+// See https://github.com/securingsincity/react-ace/issues/1233 (although there is no explanation).
+import 'ace-builds/src-noconflict/ace';
```

Comment:
To note: this thing is very weird and I don't actually know the reason why this import statement is needed for the error to not appear. My guess is that there's some weird load order issue going on, although that doesn't explain why this import fixes it.